src/App.tsx

```diff
@@ -10,6 +11,31 @@ function App() {
   const [searchFilters, setSearchFilters] = useState<string[]>([]);
   const [filterListings, setFilterListings] = useState<JobListingObj[]>([]);
 
+  // set state for mobile breakpoint for image change
+  const [belowBreakpoint, setBelowBreakpoint] = useState<boolean>(false);
+
+  // useEffect to check for breakpoint change
+  // if window.innerwidth is less than 376px then set
+  // state to true else set to false
+  useEffect(() => {
+    // if (window.innerWidth < 376) {
```

Comment:
I know it's not much code but thinking about moving this into it's own hook to clean up App a bit but also feel like this type of hook could be reused in other parts of the app (at some point) as well as other apps in general.